et --hard HEAD

```diff
@@ -0,0 +1,6679 @@
+[33mcommit 7cd57e6ecdd1fb494c4b5b51112dae803d82ee8b[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m)[m
```

Comment:
This entire file looks like it was an accidental commit. Remove it.

Reply:
It also appears to be the cause of the CI build failure, as `git ls-files` is choking on this file.